Source/WebCore/platform/graphics/cocoa/WebCoreCALayerExtras.h

```diff
@@ -52,7 +52,7 @@
 namespace WebCore {
 
 using LayerAndPoint = std::pair<CALayer *, FloatPoint>;
-WEBCORE_EXPORT void collectDescendantLayersAtPoint(Vector<LayerAndPoint, 16>& layersAtPoint, CALayer *parent, CGPoint, const std::function<bool(CALayer *, CGPoint localPoint)>& pointInLayerFunction);
+WEBCORE_EXPORT void collectDescendantLayersAtPoint(Vector<LayerAndPoint, 16>& layersAtPoint, CALayer *parent, CGPoint, const std::optional<std::function<bool(CALayer *, CGPoint localPoint)>>& pointInLayerFunction = { });
```

Comment:
`std::optional<std::function<>>` is rarely a good idea since std::function has a null state. You could just use `= nullptr` for the default value and then null-check the std::function before calling it.

Reply:
TIL that `std::function` has a null state. 👍